content/04.use-cases-b.md

```diff
@@ -85,6 +85,10 @@ Tools that link with GitHub have been developed with synchronous writing in mind
 HackMD provides a collaborative writing platform based on Markdown that integrates with GitHub.\
 We used this platform early on in the process of writing this manuscript to generate an outline.
 
+Later, we wrote the manuscript by Manubot, which is one of the important collaborative manuscript platform uses Markdown for writing and GitHub for storing and tracking changes over time [@https://doi.org/10.1371/journal.pcbi.1007128]
+
+Recently, more manuscripts were written on GitHub via Manubot(see examples <https://manubot.org/catalog/>)
+
```

Comment:
I don't know how much we want to get into specifics here.  If we mention Manubot, I feel like we should also mention [rmarkdown](https://jhemberger.github.io/posts/posts/r-markdown-manuscripts/) and [quarto](https://quarto.org/) which also use Pandoc and can be used for writing markdown manuscripts.  I'd welcome other opinions here.

Reply:
You may be right about specificity. However, I intend to mention about platform that we used to write this manuscript. We fill the GitHub toolbox of researchers with essential (or basic) tools for the Ecology and Evolution research.